### PR TITLE
ASM code for Minor Extended Sprites implementation

### PR DESCRIPTION
**Pull request by TheLX5**
Seems to work flawlessly. Check SnesLab's PIXI channel for test cases.

### CODE REVIEW

**Comment by TheLX5** [comment]
Actually, I forgot I can submit files here lol. Here you go.

[minorextended_test.zip](https://github.com/JackTheSpades/SpriteToolSuperDelux/files/6610967/minorextended_test.zip)

File: routines/SpawnMinorExtended.asm [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+;   C = Spawn status
+;       Set = Spawn failed
+;       Clear = Spawn successful
+
+?main:
+    xba 
+    lda !15A0,x
+    ora !186C,x
+    ora !15D0,x
+    ;bne ?.ret
```
Comment (Atari2) [inline routines/SpawnMinorExtended.asm:24]: Semicolon should not be here, pls fix

Comment (Atari2) [inline routines/SpawnMinorExtended.asm:24]: Semicolon should be removed.

**Review by Atari2 — CHANGES_REQUESTED** [review]

**Comment by TheLX5** [comment]
No longer relevant as it's contained in https://github.com/JackTheSpades/SpriteToolSuperDelux/pull/44

**Pull request closed without merging**